From 080d5fc27bfab4fe82b9d3aae3a5fd2609dbece0 Mon Sep 17 00:00:00 2001
From: Jianxu Zheng <jian.xu.zheng@intel.com>
Date: Wed, 11 Feb 2015 17:39:40 +0800
Subject: [PATCH 278/284] ov8858: correct acpi id

ov8858 HID is a temporary workaround.
official HID now is in place.

Tracked-On: #H1503918807
Change-Id: I64cb2569d07d24d3844aef99adfb6bb82605a1e7
Signed-off-by: Jianxu Zheng <jian.xu.zheng@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/43470
Reviewed-by: Jing Tao <jing.tao@intel.com>
Build: viefandr
Build: Wei Tao <wei.tao@intel.com>
Integrator: Wei Tao <wei.tao@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/ov8858.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.c b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.c
index 3feca1a..d154e03c 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov8858.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov8858.c
@@ -2306,7 +2306,7 @@ MODULE_DEVICE_TABLE(i2c, ov8858_id);
 
 #ifdef CONFIG_GMIN_INTEL_MID
 static struct acpi_device_id ov8858_acpi_match[] = {
-	{"INT33BF"},
+	{"INT3477"},
 	{},
 };
 #endif
-- 
1.9.1

